code/game/turfs/simulated/cleaning.dm

```diff
@@ -39,6 +39,10 @@
 		for(var/obj/effect/O in src)
 			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
 				qdel(O)
+		if(ishuman(user) && user.stats && user.stats.getPerk(/datum/perk/neat))
+			var/mob/living/carbon/human/H = user
+			if(H.sanity)
+				H.sanity.changeLevel(0.5)
```

Comment:
This code is flawed -- it doesn't activate when using cleaner spray, cleaner grenades, or even broad sweeps of mops, but only when you clean a single tile at a time (single-tile mopping, soap, wet rags, etc) and will likely significantly underwhelm any players expecting it to be a good source of sanity regain

Reply:
Additionally, it still triggers even if there's nothing to clean, so a custodian could theoretically spam their mop all over a clean floor to get their sanity back